pkg/extensions/errorhandler.go

```diff
@@ -0,0 +1,7 @@
+package extensions
+
+import "context"
```

Comment:
Linked to default handler comments.

```suggestion
import (
	"context"
	
	"github.com/lerenn/asyncapi-codegen/pkg/extensions/errorhandlers"
)
```